kit/inputs/src/UITextView/UITextView.tsx

```diff
@@ -10,15 +10,29 @@ const inputTypographyStyle = StyleSheet.flatten(Typography[textViewTypographyVar
 const textViewLineHeight = inputTypographyStyle.lineHeight ?? UILayoutConstant.smallCellHeight;
 
 /**
- * The singleline input on the ios platform has a bug -
- * if `(lineHeight - fontSize) > 5` it starts to crop
- * letters from the bottom (letters like "g", "j").
- * Style prop `textAlignVertical` works only on Android.
+ * Android TextInput does not respond to lineHeight changes,
```

Comment:
This is probably not working well on all Android devices...

Reply:
I didn't notice anything wrong. I think we should test it, and fix problems only if they found.